### PR TITLE
General enhancement of docs, add cli documentation

### LINKED ISSUE #160: Install instruction
It would be really helpful (for those at the bottom of the learning curve) if there were a couple of lines in the README on how to install this software.  
"See bimmerconnected for usage instruction" is not very helpful.  What does it mean?

### PR DESCRIPTION
**Pull request by rikroe**
Fixes #160.

Through the issue I had another view at our docs. This resulted in some changes to make them easier to read and contain more information: 

- Change theme from `alabaster` to `sphinx_rtd_theme`
- Include a documentation for `cli`
- For the automated parsing of `cli`, the `ArgumentParser` itself had to be refactored into its own function
- Improve navigation on the docs
- Improve `README.rst` to better integrate into docs and provide better information in GitHub

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -53,18 +73,23 @@ If you want to contribute your data, perform the following steps:
 
 This will create a set of log files in the "vehicle_fingerprint" folder.
 Before sending the data to anyone please **check for any personal data**.
+
 The following attributes should be replaced with default values:
```
Comment (gerard33) [inline README.rst:77]: The vin, lat, lon, heading and license plate are automatically anomized by the code -> https://github.com/bimmerconnected/bimmer_connected/blob/184797cb4dfa15ead4b3bf0107b2bdacf95a0abe/bimmer_connected/account.py#L194.
So only need to check if there is other data they don't want to be send, like for instance the dealer info.

File: README.rst [hunk 2]
```diff
@@ -11,31 +11,51 @@
...
 the Connected Drive portal.
 
-See :code:`bimmerconnected` for usage instruction or the
-`API Documentation <http://bimmer-connected.readthedocs.io/en/latest/>`_.
 
-This library is written to include it in Home Assistant.
+Installation
+============
+Just install the latest release from `PyPI <https://pypi.org/project/bimmer-connected/>`_ 
+using :code:`pip install bimmer_connected`.
```
[the remark this answers is not in the document]
Comment (rikroe) [reply README.rst:25]: Will change 👍

File: README.rst [hunk 3]
```diff
@@ -11,31 +11,51 @@
...
+
+Alteratively, clone the project and execute :code:`pip install .` to install the current 
+:code:`master` branch.
```
[the remark this answers is not in the document]
Comment (rikroe) [reply README.rst:28]: I would not specify more (googling github clone should help 😄). Also, I would not mention any other Python-specific stuff like virtual environments etc.

File: README.rst (hunk 1 above)
Comment (rikroe) [reply README.rst]: Clarified this in the readme :)

**Comment by rikroe** [comment]
Just came to my mind: depending on the order of merging, this PR or #173 will require manual merges. 
Depending on which PR is merged first, I'll update the second accordingly.

**Comment by gerard33** [comment]
LGTM. Approved this PR so you can choose which PR to merge first.
I will review #173 now.

**Pull request merged**